src/chainstate/stacks/db/unconfirmed.rs

```diff
@@ -217,6 +241,7 @@ impl UnconfirmedState {
                     }
                     total as u64
                 };
+                self.bytes_so_far += new_bytes;
```

Comment:
Can you remove this, and put it back where it was?  I'm intentionally trying to _avoid_ mutating `self` here.

Reply:
Oh I see. The reason I changed it is because `bytes_so_far` was only adding the number of new bytes from the last microblock. 

I could keep track of a total new bytes and add that to `bytes_so_far` below.

Reply:
Yes, I believe there's a bug there as well (that should be fixed in 2.0.11.0.0-rc4).  The `new_bytes` variable should represent the total number of bytes added to the unconfirmed state.